warehouse/upload.go

```diff
@@ -846,33 +850,42 @@ func (job *UploadJobT) createLoadFiles() (loadFileIDs []int64, err error) {
 		if err != nil {
 			panic(err)
 		}
-
-		responses := <-ch
-		pkgLogger.Infof("[WH]: Received responses for staging files %d:%d for %s:%s from PgNotifier", stagingFiles[i].ID, stagingFiles[j-1].ID, destType, destID)
-		for _, resp := range responses {
-			// TODO: make it aborted
-			if resp.Status == "aborted" {
-				pkgLogger.Errorf("[WH]: Error in genrating load files: %v", resp.Error)
-				continue
-			}
-			var payload map[string]interface{}
-			err = json.Unmarshal(resp.Payload, &payload)
-			if err != nil {
-				panic(err)
-			}
-			respIDs, ok := payload["LoadFileIDs"].([]interface{})
-			if !ok {
-				pkgLogger.Errorf("[WH]: No LoadFileIDS returned by wh worker")
-				continue
-			}
-			ids := make([]int64, len(respIDs))
-			for i := range respIDs {
-				ids[i] = int64(respIDs[i].(float64))
+		wg.Add(1)
+		batchStartNumber := i
```

Comment:
Let's call this Index instead of Number? Instead of batch, may be just `startIdx` and `endIdx`